front/src/App.js

```diff
@@ -116,7 +116,7 @@ class App extends React.Component {
                     <p className="flex:1">hello from /events</p>
                   </Route>
                   <Route exact path="/todos">
-                    <p className="flex:1">hello from /todos</p>
+                    <Route path="/todos" component={Todos}/>
```

Comment:
instead of:

    <Route path="/todos" component={Todos}/>

try:

    <Todos />

remember to keep a div with the class "flex:1" ;)